README.md

```diff
@@ -16,13 +16,13 @@ MapboxStatic.swift pairs well with [MapboxDirections.swift](https://github.com/m
 Specify the following dependency in your [Carthage](https://github.com/Carthage/Carthage/) Cartfile:
 
 ```sh
-github "Mapbox/MapboxStatic.swift" "~> 0.7"
+github "Mapbox/MapboxStatic.swift" "master"
```

Comment:
For now. Whenever we rewrite the readme, we have to instruct the developer to get the master version so that the instructions are consistent. Otherwise we'd have to stage changes to the documentation, which would render them inconsistent with the rest of the repository in the meantime.

This shouldn't be a problem, because both CocoaPods and Carthage instruct developers to check in their lock files.